### PR TITLE
Bugfix: Finding ignored tests

### PR DESCRIPTION
**Pull request by vlisianskii**

### CODE REVIEW

File: engine/pom.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <parent>
         <groupId>com.github.jpmorganchase.sandboni</groupId>
         <artifactId>sandboni-core</artifactId>
-        <version>1.1.2-SNAPSHOT</version>
+        <version>1.1.4-SNAPSHOT</version>
```
Comment (wmndev) [inline engine/pom.xml]: where is 1.1.3 ?

File: pom.xml [hunk 2]
```diff
@@ -2,7 +2,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.github.jpmorganchase.sandboni</groupId>
     <artifactId>sandboni-core</artifactId>
-    <version>1.1.2-SNAPSHOT</version>
+    <version>1.1.4-SNAPSHOT</version>
```
Comment (wmndev) [inline pom.xml]: 1.1.3?

File: scm/pom.xml [hunk 3]
```diff
@@ -4,7 +4,7 @@
     <parent>
         <groupId>com.github.jpmorganchase.sandboni</groupId>
         <artifactId>sandboni-core</artifactId>
-        <version>1.1.2-SNAPSHOT</version>
+        <version>1.1.4-SNAPSHOT</version>
```
Comment (wmndev) [inline scm/pom.xml]: 1.1.3?

File: engine/pom.xml (hunk 1 above)
Comment (vlisianskii) [reply engine/pom.xml]: There is 1.1.3 version:
https://search.maven.org/artifact/com.github.jpmorganchase.sandboni/sandboni-scm/1.1.3

File: scm/pom.xml (hunk 3 above)
Comment (vlisianskii) [reply scm/pom.xml]: There is 1.1.3 version:
https://search.maven.org/artifact/com.github.jpmorganchase.sandboni/sandboni-scm/1.1.3

File: pom.xml (hunk 2 above)
Comment (vlisianskii) [reply pom.xml]: There is 1.1.3 version:
https://search.maven.org/artifact/com.github.jpmorganchase.sandboni/sandboni-scm/1.1.3

**Pull request merged**